src/__tests__/queryByApi.test.js

```diff
@@ -48,7 +48,7 @@ test('queryByText not found', () => {
   ).toBeFalsy();
 });
 
-test('queryByText nested multiple <Text> in <Text>', () => {
+test('queryByText nested text across multiple <Text> in <Text>', () => {
   expect(
     render(
       <Text>
```

Comment:
Could we add similar `nativeID` check for this test too? IMO it should return top level `Text`.